packages/toast/docs.mdx

```diff
@@ -0,0 +1,23 @@
+---
+componentName: 'Toast'
```

Comment:
```suggestion
componentName: 'Toast'
peerComponents:
  - 'Notification'
```

Reply:
(this would require `Notification` to have a `docs.mdx` file)

Reply:
Looks to be working without needing to add peerComponents.